pivotTable.go

```diff
@@ -785,15 +785,14 @@ func (f *File) getPivotTableDataRange(opts *PivotTableOptions) error {
 		opts.pivotDataRange = opts.DataRange
 		return nil
 	}
-	for _, sheetName := range f.GetSheetList() {
-		tables, err := f.GetTables(sheetName)
-		e := ErrSheetNotExist{sheetName}
-		if err != nil && err.Error() != newNotWorksheetError(sheetName).Error() && err.Error() != e.Error() {
-			return err
-		}
-		for _, table := range tables {
-			if table.Name == opts.DataRange {
-				opts.pivotDataRange, opts.namedDataRange = fmt.Sprintf("%s!%s", sheetName, table.Range), true
+	tbls, err := f.getTables()
+	if err != nil {
+		return err
+	}
+	for sheetName, tables := range tbls {
```

Comment:
There're some blank prefix characters in these lines, please format source code by `gofmt`.

Reply:
Done.